src/core/uint.js

```diff
@@ -217,16 +216,7 @@ UInt.prototype.parse_hex = function(j) {
 };
 
 UInt.prototype.parse_bits = function(j) {
-  if (sjcl.bitArray.bitLength(j) === this.constructor.width * 8) {
-    const bytes = sjcl.codec.bytes.fromBits(j);
-    this.parse_bytes(bytes);
-  } else {
-    this._value = NaN;
-  }
-
-  this._update();
-
-  return this;
+  return this.parse_bytes(sjclcodec.bytes.fromBits(j));
```

Comment:
Other than these bitty *bits routines, all we depend upon in sjcl is the utf8 codec really. We could just cull these, and find some other utf8 codec. I'm leaning ever so lightly towards just keeping these though.